sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1651,6 +1652,40 @@ export class ApplicationService extends RecordDataModelService<Application> {
     }
   }
 
+  /**
+   * Gets application and assessment status details.
+   * @param applicationId application id.
+   */
+  async getApplicationAssessmentStatusDetails(
+    applicationId: number,
+  ): Promise<ApplicationAssessmentStatusDetailsAPIOutDTO> {
+    const application = await this.repo.findOne({
```

Comment:
Looks like this service can be re-used under create manual reassessments to get application details.